spec/commands/foreman/foreman_command_spec.rb

```diff
@@ -1,4 +1,31 @@
 require 'commands/foreman/foreman_command'
 
 RSpec.describe ForemanCommand do
+  let (:foreman) { ForemanCommand.new }
+
+  it "applies to a foreman command" do
+    expect(foreman.applies_to("foreman")).to be true
+  end
+
+  it "freman comman isn't case sensitive" do
+    expect(foreman.applies_to("Foreman")).to be true
+  end
+
+  it "freman comman isn't spaces sensitive" do
+    expect(foreman.applies_to("Foreman")).to be true
```

Comment:
That's the same test as the previous one. According to the description I think you wanted to add some spaces here?